package.json

```diff
@@ -1,12 +1,22 @@
 {
   "name": "flux",
+  "version": "0.0.1",
+  "publisher": "influxdata",
   "displayName": "Flux for VSCode",
   "description": "Flux language extension for VSCode",
-  "version": "0.0.1",
+  "author": {
+    "name": "InfluxData"
+  },
   "icon": "https://github.com/influxdata/vsflux/images/icon.png",
-  "author": "InfluxData",
-  "repository": "github:influxdata/vsflux",
+  "categories": [
+    "Programming Languages",
+    "Snippets"
+  ],
   "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/influxdata/vsflux"
+  },
```

Comment:
let's keep using this format `"repository": "github:user/repo"` described in https://docs.npmjs.com/files/package.json

Reply:
I feel like the shorthands are less readable, but I can add them if you think it's necessary